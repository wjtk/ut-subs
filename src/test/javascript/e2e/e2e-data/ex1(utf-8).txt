0:10
zażółć gęślą jaźń
0:11
0:12
jeden dwa
