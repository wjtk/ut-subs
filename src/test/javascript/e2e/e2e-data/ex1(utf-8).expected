00:00:10:zażółć gęślą jaźń
00:00:12:jeden dwa
